### ISSUE TITLE
rmdup_mvicuna_bam fails

### ISSUE THREAD

**Issue by haydenm** [broadinstitute/viral-ngs#425]
Running `read_utils::rmdup_mvicuna_bam` fails. It says

```
read_utils:840:rmdup_mvicuna_bam - WARNING - no reads found in /tmp/....../AP398.1.
TAAGGCGA-TAGATCGC_1.fastq,
                                    assuming that's because there's no reads in that read group
```

For an example of a file on which is fails, see `/idi/sabeti-scratch/hmetsky/viral-work/results/viral-ngs/zika/clinical-samples/zika_brazil_fiocruz-thiago/data/00_raw/B4-HS-hiseq.bam`. Rolling back to 0dc7ab21e fixes this. @tomkinsc noted that issue may be related to an updated Picard that changes `SamToFastq`.

**Comment by dpark01** [comment]
Wait, but where does it actually fail? (can you add a stack trace?) that particular warning is intended to be there but not fail.

**Comment by haydenm** [comment]
It doesn't technically "fail"/crash. It just outputs a BAM with 0 reads.

**Comment by dpark01** [comment]
nuts.. okay lemme take a quick look.

**Comment by haydenm** [comment]
Oh, @tomkinsc is on it! He just asked me to make this issue for reference.

**Comment by dpark01** [comment]
ok cool great! we probably just need to update `tools.picard.SamToFastqTool().per_read_group` to the new behavior

**Comment by tomkinsc** [comment]
WIP, hopefully addressed by #426.
